src/Server/modules/DataClasses.js

```diff
@@ -84,7 +84,9 @@ module.exports = {
 				if (!Number.isNaN(dates[i].getMonth()) && !Number.isNaN(parseInt(values[i]))) {
 					this.numDeaths[dates[i]] = parseInt(values[i]);
 				}
+				this.currentNumDeaths = this.numDeaths[dates[i]];
```

Comment:
We want currentNumDeaths = the number of cases for the most recent date. Therefore, we do not want to assign currentNumDeaths with this.numDeaths[dates[i]].

Reply:
so what can we do since date is not sorted?